app/components/ui/forms.tsx

```diff
@@ -98,11 +98,26 @@ export function Button({ children, ...props }: ButtonProps) {
 type LinkButtonProps = {
   children: ReactNode;
   to: string;
+  preventScrollReset?: boolean;
+  prefetch?: LinkProps['prefetch'];
 };
 
-export function LinkButton({ to, children, ...props }: LinkButtonProps) {
+export function LinkButton({
+  to,
+  children,
+  preventScrollReset = true,
+  prefetch = 'intent',
+  ...props
+}: LinkButtonProps) {
   return (
-    <Link {...props} to={to} className="border-2 bg-red-700 text-white rounded-lg p-3 w-full text-center" type="submit">
+    <Link
+      {...props}
+      to={to}
+      prefetch="intent"
+      className="border-2 bg-red-700 text-white rounded-lg p-3 w-full text-center"
+      preventScrollReset={preventScrollReset}
+      type="submit"
```

Comment:
Since we are rendering an anchor tag here, we can remove `type="submit"` - not applicable.